cmd/babylond/cmd/flags.go

```diff
@@ -19,76 +19,78 @@ import (
-	flagGenesisTime                = "genesis-time"
-	flagBlockGasLimit              = "block-gas-limit"
-	flagVoteExtensionEnableHeight  = "vote-extension-enable-height"
-	flagCovenantPks                = "covenant-pks"
-	flagCovenantQuorum             = "covenant-quorum"
-	flagMinStakingAmtSat           = "min-staking-amount-sat"
-	flagMaxStakingAmtSat           = "max-staking-amount-sat"
-	flagMinStakingTimeBlocks       = "min-staking-time-blocks"
-	flagMaxStakingTimeBlocks       = "max-staking-time-blocks"
-	flagMaxActiveFinalityProviders = "max-active-finality-providers"
-	flagMinUnbondingTime           = "min-unbonding-time"
-	flagUnbondingFeeSat            = "unbonding-fee-sat"
-	flagSlashingPkScript           = "slashing-pk-script"
-	flagMinSlashingFee             = "min-slashing-fee-sat"
-	flagSlashingRate               = "slashing-rate"
-	flagMinCommissionRate          = "min-commission-rate"
-	flagSignedBlocksWindow         = "signed-blocks-window"
-	flagMinSignedPerWindow         = "min-signed-per-window"
-	flagFinalitySigTimeout         = "finality-sig-timeout"
-	flagJailDuration               = "jail-duration"
+	flagMaxActiveValidators           = "max-active-validators"
+	flagBtcConfirmationDepth          = "btc-confirmation-depth"
+	flagEpochInterval                 = "epoch-interval"
+	flagBtcFinalizationTimeout        = "btc-finalization-timeout"
+	flagCheckpointTag                 = "checkpoint-tag"
+	flagBaseBtcHeaderHex              = "btc-base-header"
+	flagBaseBtcHeaderHeight           = "btc-base-header-height"
+	flagAllowedReporterAddresses      = "allowed-reporter-addresses"
+	flagInflationRateChange           = "inflation-rate-change"
+	flagInflationMax                  = "inflation-max"
+	flagInflationMin                  = "inflation-min"
+	flagGoalBonded                    = "goal-bonded"
+	flagBlocksPerYear                 = "blocks-per-year"
+	flagGenesisTime                   = "genesis-time"
+	flagBlockGasLimit                 = "block-gas-limit"
+	flagVoteExtensionEnableHeight     = "vote-extension-enable-height"
+	flagCovenantPks                   = "covenant-pks"
+	flagCovenantQuorum                = "covenant-quorum"
+	flagMinStakingAmtSat              = "min-staking-amount-sat"
+	flagMaxStakingAmtSat              = "max-staking-amount-sat"
+	flagMinStakingTimeBlocks          = "min-staking-time-blocks"
+	flagMaxStakingTimeBlocks          = "max-staking-time-blocks"
+	flagMaxActiveFinalityProviders    = "max-active-finality-providers"
+	flagMinUnbondingTime              = "min-unbonding-time"
+	flagUnbondingFeeSat               = "unbonding-fee-sat"
+	flagSlashingPkScript              = "slashing-pk-script"
+	flagMinSlashingFee                = "min-slashing-fee-sat"
+	flagSlashingRate                  = "slashing-rate"
+	flagMinCommissionRate             = "min-commission-rate"
+	flagSignedBlocksWindow            = "signed-blocks-window"
+	flagFinalityActivationBlockHeight = "finality-activation-block-height"
```

Comment:
```suggestion
	flagActivationHeight = "activation-height"
```

try to be as much succinct as possible for parameters